new/LIT7111Zodiac.xml

```diff
@@ -5,11 +5,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title xml:lang="en" xml:id="t1">Treatise on the signs of the zodiac</title>
+                <title xml:lang="en" xml:id="t1">Astrological treatise on the twelve signs of the zodiac</title>
+                <title xml:lang="gez" xml:id="t2">ሐሳበ፡ ከዋክብት፡</title>
+                <title xml:lang="gez" corresp="#t2" type="normalized">Ḥassāba kawākǝbt</title>
+                <title xml:lang="gez" xml:id="t3">ፍካሬ፡ ከዋክብት፡</title>
+                <title xml:lang="gez" corresp="#t3" type="normalized">Fǝkkāre kawākǝbt</title>
+                <title xml:lang="gez" xml:id="t4">ፍካሬሆሙ፡ ለ፲፪ ከዋክብት፡</title>
                 <editor role="generalEditor" key="AB"/>
                 <editor key="MV"/>
+                <editor key="DE"></editor>
                 <funder>Akademie der Wissenschaften in Hamburg</funder>
             </titleStmt>
+            <editionStmt>
+                <p>Work description funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) – Project number 536604808) <ptr target="https://gepris.dfg.de/gepris/projekt/536604808"/>.</p>
```

Comment:
it should be `ref target`, not ptr (wrong in Demarginalizing files, am fixing now)

You might want to change the brackets (one too many closing brackets?)

You might want to add more information on the project to make it more visible
as you have seen Demarginalizing wording was
```
<editionStmt>
<p>
Cataloguing funded by the AHRC-DFG Project 'Demarginalizing medieval Africa: Images, texts, and identity in early Solomonic Ethiopia (1270-1527)', grant ref. nos. AH/V002910/1 - 448410109
<ref target="https://gepris.dfg.de/gepris/projekt/448410109"/>
</p>
</editionStmt>
```
so with the project name